collect_app/src/androidTest/java/org/odk/collect/android/feature/entitymanagement/ViewEntitiesTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.odk.collect.android.feature.entitymanagement
+
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.odk.collect.android.support.TestDependencies
+import org.odk.collect.android.support.pages.FormEntryPage
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain
+
+class ViewEntitiesTest {
+
+    private val rule = CollectTestRule(useDemoProject = false)
+    private val testDependencies = TestDependencies()
+
+    @get:Rule
+    val ruleChain: RuleChain = TestRuleChain.chain(testDependencies)
+        .around(rule)
+
+    @Test
+    fun canViewLocallyCreatedEntitiesInBrowser() {
```

Comment:
I think we started using different names... online and offline entities are I think the clearest options but here we have `local entities`. The test title below is also not clear it's about online entities.

Reply:
The reason I ended up using local is because we know store both the "offline" and "online" entities in one place ("local"), but can create or update them locally (rather than those creates/updates coming from the server). It's definitely got me a little twisted up. Do you think we should use "offline" here? 

I reckon this will become easier again with #6011, as we'll need to start tracking if local entities are "offline" or "online" so we know whether to delete them or not if they're not in the "online" lists any longer.

Reply:
Could we leave this for #6011? I think, it'll be best to consolidate the naming there.